# Lily_Bioinformatics

This is a collection of links to repositories holding code / instructions developed for the Lily Precision Diagnostics project

## WGS in the UCL HPC environment

[MitoSalt for deletions and duplications](https://github.com/rkabiljo/MitoSalt_UCL_Cluster)

[experimenting with extraction of deletion features from MitoSalt output](https://github.com/rkabiljo/deletionParameters)

[MitoHPC2 - works on WGS and LongReads](https://github.com/rkabiljo/MitoHPC2_LongReads_UCL)

[Retrieve and update panels from Genomics England PanelApp](https://github.com/tamjhill/GE_panel_scripts)

[Tamara's ICGNMD pipelines](https://github.com/tamjhill/ICGNMD_project)

[run GATK pipeline to align reads and call variants](https://github.com/rkabiljo/GATK_Myriad) - largely replaced by Tamara's ICGNMD

[apply panels in the UCL HPC environent](https://github.com/rkabiljo/applyPanel_UCL_cluster) - laregely replaced by agnostic (under Long Reads)



## WGS in the GeL environment
[apply panels to cohorts of patients in GeL](https://github.com/rkabiljo/applyPanelToCohort)

[get deNovo variants from main release trios](https://github.com/rkabiljo/exploreDeNovo_GeL)

[Small Variant Workflow - run and process results](https://github.com/tamjhill/GeL_small_variant_workflow/)

## Long Read Sequencing
[PacBio RNASeq data processing, IsoSeq](https://github.com/rkabiljo/RNASeq_PacBio)

[downloading, basecalling, and epi2me on ONT data](https://github.com/rkabiljo/LR_ONT_UCL)

[MitoHPC2 on Long Read data, both PacBio and ONT](https://github.com/rkabiljo/MitoHPC2_LongReads_UCL)

[phase two mitochondrial variants](https://github.com/rkabiljo/phaseMitoVariants)

[PacBio align and call variants](https://github.com/tamjhill/pacbioAlignerCaller)

[SVs in PacBio, calling and analysis](https://github.com/tamjhill/longread_sv_cnvs)

[Apply Panels and Run Agnostic on the variants called by PacBio human variation wdl OLD](https://github.com/rkabiljo/PacBio_Panels_and_Agnostic)

[Agnostic Pipeline with Annotated Panels: NEW](https://github.com/rkabiljo/agnostic_pipeline)

[Methylation data: From Long Read BAMs to Epigen Centralsubmission files](https://github.com/rkabiljo/MethylationForEpigenCentral/)

## Long Reads - Deletions and Duplications

[MitoDetective:for PacBio and ONT, calls a single deletion or duplication, will add more](https://github.com/rkabiljo/mitoDetective)

[Useful snippets of code to examine unusual reads on chrM](https://github.com/rkabiljo/duplication_exploration)

[As above, but focussed on these very long reads and examining where they come from](https://github.com/rkabiljo/extraLongMito)

[Analyse and cluster deletions in Long Reads](https://github.com/tamjhill/deletions_duplications)



## General information for UCL HPC and RDS
[General information paths, copying, connecting on UCL HPC](https://github.com/rkabiljo/UCL_HPC_settings)




## RNASeq, short reaads

[RNASeq on short reads - includind differential transcripts, genes, etc](https://github.com/rkabiljo/RNASeq_shortReads)

[RNASeq Majiq and DROP: Salmon,Majiq](https://github.com/rkabiljo/RNASeq_UCL)

[RNASeq in GeL with GeL pilot](https://github.com/rkabiljo/RNSeqInGeL)

## Work in progress or not used
[Acollection of scripts to programatically access GeL panel app APIs](https://github.com/rkabiljo/panelAppWork)

[MitoHPC on UCL Kathleen, moved now to Myriad](https://github.com/rkabiljo/MitoHPCOnUCL)

[MitoSalt on UCL Kathleen, moved now to Myriad](https://github.com/rkabiljo/MitoSALtOnKathleen)
