# Lily_Bioinformatics

This is a collection of links to repositories holding code / instructions developed for the Lily Precision Diagnostics project

## WGS in the UCL HPC environment

[run GATK pipeline to align reads and call variants](https://github.com/rkabiljo/GATK_Myriad)

[apply panels in the UCL HPC environent](https://github.com/rkabiljo/applyPanel_UCL_cluster)

[experimenting with extraction of deletion features from MitoSalt output](https://github.com/rkabiljo/deletionParameters)


## WGS in the GeL environment
[apply panels to cohorts of patients in GeL](https://github.com/rkabiljo/applyPanelToCohort)

## Long Read Sequencing

[downloading, basecalling, and epi2me on ONT data](https://github.com/rkabiljo/LR_ONT_UCL)

[MitoHPC2 on Long Read data, both PacBio and ONT](https://github.com/rkabiljo/MitoHPC2_LongReads_UCL)
[phase two mitochondrial variants](https://github.com/rkabiljo/phaseMitoVariants)


## General information for UCL HPC and RDS
[General information paths, copying, connecting on UCL HPC](https://github.com/rkabiljo/KathleenAndMyriad)

## RNASeq, short reaads, UCL HPC
[RNASeq processing on UCL HPC: Salmon,Majiq](https://github.com/rkabiljo/RNASeq_UCL)

[RNASeq work done for Micol - includind differential transcripts, genes, etc](https://github.com/rkabiljo/RNASeq_Micol)

## Work in progress or not used
[Acollection of scripts to programatically access GeL panel app APIs](https://github.com/rkabiljo/panelAppWork)
[MitoHPC on UCL Kathleen, moved now to Myriad](https://github.com/rkabiljo/MitoHPCOnUCL)
[MitoSalt on UCL Kathleen, moved now to Myriad] (https://github.com/rkabiljo/MitoSALtOnKathleen)
